src/placeos-driver/proxy/scheduler.cr

```diff
@@ -57,29 +68,29 @@ class PlaceOS::Driver::Proxy::Scheduler
     spawn(same_thread: true) { run_now(block) } if immediate
     wrapped = nil
     task = Tasker.in(time) do
-      @schedules.delete(wrapped.not_nil!.object_id)
+      @schedules.delete(wrapped.not_nil!)
```

Comment:
```suggestion
      @schedules.delete(wrapped) if wrapped
```

Reply:
You can even drop that check completely. `Set#delete` is not bound by the type of the set.